src/components/Link/Link.scss

```diff
@@ -39,22 +44,9 @@
   @include ms-Link;
 }
 
-// TODO: Remove overrides below.
```

Comment:
Ah, good catch--yes, that would definitely break that scenario. @dubunin--why is this being removed?